impl/client/src/main/javascript/web/prompting/PromptPanel.js

```diff
@@ -1405,7 +1405,7 @@ function(Base, Logger, DojoNumber, i18n, Utils, GUIDHelper, WidgetBuilder, Dashb
 
         // Must submit, independently of auto-submit value.
         // All parameters are initialized, fire the submit
-        fireSubmit = !noAutoAutoSubmit;
+        fireSubmit = (noAutoAutoSubmit != undefined) ? !noAutoAutoSubmit : false;
```

Comment:
Maybe it's clearer this way? Also, when comparing with `==` (or `!=`) it's best (and equivalent) to compare with `null`.
```suggestion
        fireSubmit = (noAutoAutoSubmit != null) && !noAutoAutoSubmit;
```